Makefile

```
@@ -247,6 +262,20 @@ demo-data:
 demo-data-interactive:
 	@$(MAKE) demo-data DEMO_NON_INTERACTIVE=0
 
+.PHONY: examples-test
+
+# examples-test builds every example program under examples/ and runs
+# the test suite for examples that ship one (notably 09-testing-with-mocks).
+# It is the compile-time guarantee that every example tracks the public SDK
+# surface — refactors that break a documented call shape break the build here.
+examples-test:
```

checkmake (0.3.2)

[warning] Target body for "examples-test" exceeds allowed length of 5 lines (6).

(maxbodylength)